package.json

```diff
@@ -24,6 +24,7 @@
     "lint:sol-logic": "solhint -c .solhint.json 'src/**/*.sol' 'script/**/*.sol'",
     "lint:sol-tests": "solhint -c .solhint.tests.json 'test/**/*.sol'",
     "prepare": "husky install",
+    "smock": "smock-foundry --contracts src/contracts",
```

Comment:
we should add this when building and running the tests

Reply:
mhhh, i've added it on the CI, but i believe smock contracts are sth that are not gonna suffer much changes, and if it does all you get is a compilation error on a smock contract, and you'll know you must run `yarn smock`, but adding this routine to the `yarn build` seems like suboptimal time-wise, wdyt?

Reply:
no worries, I was suggesting it so you can run `yarn test:unit` without having to run `yarn smock` before